using_the_api.md

```diff
@@ -606,6 +607,11 @@ This is a special type of autocomplete query which allows users to search for ge
     * `domestic` - Matches only entries with the United States as the `location_country_code`
     * `foreign` - Matches only entries where the `location_country_code` is _not_ the United States
     * `all` - Matches any location entry. This is the default behavior
+  * `usage` - _Optional_ - The usage of the search. Options available are:
+    * `recipient` - Matches only entries where the location is used as a recipient location
+    * `place_of_performance` - Matches only entries where the location is used as a place of performance
+    * `both` - Matches only entries that serve as both recipient location and place of performance
```

Comment:
Ah, this answers the `both/all` question I had in `api_request_utils`. So my new question: why do we need a both option if we have an all option?